tests/lib/journalctl.sh

```diff
@@ -12,12 +15,23 @@ start_new_journalctl_log(){
         echo "Empty journalctl cursor, exiting..."
         exit 1
     else
-        echo "$cursor" > "$JOURNALCTL_CURSOR_FILE"
+        echo "$SPREAD_JOB " >> "$JOURNALCTL_CURSOR_FILE"
+        echo "$cursor" >> "$JOURNALCTL_CURSOR_FILE"
+    fi
+
+    echo "New test starts here - $SPREAD_JOB" | systemd-cat
+    test_id="test-${RANDOM}${RANDOM}"
```

Comment:
`${RANDOM}` is bash specific. Not a problem though as we already have `#!/bin/bash` at the top, just wanted to highlight that this is introducing another bashism.